html/pfappserver/root/static/js/node.js

```diff
@@ -421,6 +429,41 @@ NodeView.prototype.searchPagination = function(e) {
     return false;
 };
 
+NodeView.prototype.refreshPage = function() {
+    var that = this;
+    var pagination = $('.pagination').first();
+    var formId = pagination.attr('data-from-from') || '#search';
+    var form = $(formId);
+    var link = pagination.find('li.disabled a').first();
+    if(form.length == 0) {
+        form = $('#search');
+    }
+    var columns = $('#columns');
+    var href = link.attr("href");
+    var section = $('#section');
+    var status_container = $("#section").find('h2').first();
+    var loader = section.prev('.loader');
+    loader.show();
+    section.fadeTo('fast', 0.5);
+    section.fadeTo('fast', 0.5, function() {
+        that.nodes.post({
+            url: href,
+            data: form.serialize() + "&" + columns.serialize(),
+            always: function() {
+                loader.hide();
+                section.fadeTo('fast', 1.0);
+            },
+            success: function(data) {
+                section.html(data);
+            },
+            errorSibling: status_container
+        });
+    });
+    return false;
+};
+
+
```

Comment:
There are 3 lines separating both methods. Can you put it to 1 like the other methods.